### PR TITLE
[sqlite] Updated to 3.32.1 to fix a security vulnerability

### PR DESCRIPTION
**Pull request by xonix-22**
Security vulnerability highlighted during SVM

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

File: ports/sqlite3/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: sqlite3
-Version: 3.32.0-0
+Version: 3.32.1-0
```
Comment (NancyLi1013) [inline ports/sqlite3/CONTROL:2]:
```suggestion
Version: 3.32.1
```
The version should be 3.32.1.

**Comment by NancyLi1013** [comment]
Feature has passed with the following triplets:
- x86-windows
- x64-windows
- x64-windows-static

**Comment by dan-shaw** [comment]
Thanks for the PR!

**Pull request merged**